src/components/layer-controller/LayerController.js

```diff
@@ -254,21 +254,19 @@ export default function LayerController(props) {
     );
   }
 
-  const classes = useExpansionPanelStyles();
-  const summaryClasses = useExpansionPanelSummaryStyles();
+  const controllerSectionClasses = useControllerSectionStyles();
   const closedOpacityLabelClasses = useSmallInputLabelStyles();
```

Comment:
The `closedOpacityLabelClasses` are now ignored / take lower precedence, so when the panel is closed the font size is `1rem` rather than the `14px` specified by `useSmallInputLabelStyles`.

This PR also introduces extra padding on the bottom of the closed panel, were these style changes intended?
<img width="331" alt="Screen Shot 2021-06-01 at 11 29 08 AM" src="https://user-images.githubusercontent.com/7525285/120350475-03a28980-c2cd-11eb-9838-1172681f5258.png">

<img width="329" alt="Screen Shot 2021-06-01 at 11 29 01 AM" src="https://user-images.githubusercontent.com/7525285/120350499-0604e380-c2cd-11eb-8887-8cb8f99b80a8.png">

<img width="426" alt="Screen Shot 2021-06-01 at 11 28 38 AM" src="https://user-images.githubusercontent.com/7525285/120350511-08ffd400-c2cd-11eb-8a8b-1aa590b3ac0f.png">